### ISSUE TITLE
Versioning pain?

### ISSUE THREAD

**Issue by AlexMouton** [purescript-contrib/purescript-arraybuffer-types#6]
Hey Gary
Did you mean to increase the version on this from 1.0 on april 2 to 2.0 on april 17?
This seems to cause some grief in installing projects around the purescript world as some now depend on ^1.0.0 and others ^2.0.0 and bower gives up: ECONFLICT Unable to find suitable version for purescript-arraybuffer-types

2.0: https://github.com/purescript-node/purescript-node-http/blob/master/bower.json
1.0: https://github.com/slamdata/purescript-affjax/blob/master/bower.json

Thanks
Alex

**Comment by garyb** [comment]
It wasn't me that made the v2 release actually - I submitted the PR with the breaking change but hadn't necessarily meant to release it immediately, since the change is breaking but wasn't super important, so it could have waited for a future round of breaking changes to the libraries.

There is a change between v1 and v2, since a new `ArrayViewType` kind was introduced and the `ArrayView` type was updated to use it. It probably doesn't affect many (or maybe any) dependants due to the way `ArrayView` is commonly used, but there is no way to guarantee that for sure, so the breaking bump is correct.

I'd suggest adding a resolution for now, to choose one or the other... either will probably work!
